doc/getting-started.md

```diff
@@ -795,27 +777,42 @@ Open [Bitbucket](http://192.168.56.31:7990/), login with your crowd administrati
 Here open the User section. If you can't see the CD user, you have to synchronize the Crowd directory in the **User directories** section.
 Click on the CD user. In the user details you have the possiblity to add a SSH key. Click on the tab and enter the _public key_ from the generated key pair.
 
+### Setup and configure Sonarqube
+
+Amend `ods-configuration/ods-core/sonarqube/ocp-config/sonarqube.env`
+and type
+
+``` bash
+tailor update
+
+``` 
+confirm with `y` and installation should start.
+
+After th installation has taken place, change to the Openshift Webconsole and start a sonarqube build. 
```

Comment:
`s/th/the`
`s/Openshift/OpenShift`
`s/sonarqube/SonarQube`

Reply:
@michaelsauter fixed. Would you please approve again?